src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectMessage.cs

```diff
@@ -280,6 +304,20 @@ public string DomainHint
         }
 
         /// <summary>
+        /// Gets or sets whether parameters for the library and version are sent on the query string for this <see cref="OpenIdConnectMessage"/> instance. 
+        /// This value is set to the value of EnableTelemetryParametersByDefault at message creation time.
+        /// </summary>
+        public bool EnableTelemetryParameters
```

Comment:
@polita I don't see this in current PR. I was thinking this should be on the Property EnableTelemetryParameters